jupyter-org-client.el

```diff
@@ -284,6 +284,8 @@ to."
         ;; TODO: Handle all of the different macro types for inline results, see
         ;; `org-babel-insert-result'.
         (setq data `(:text/plain ,(plist-get data :text/plain)))
+        ;; NOTE(tor): This is where the magic happens. Could potentially
```

Comment:
This comment doesn't seem related to the pull request feature, at least I can't tell what it is referring to. Can you remove it or comment more how this would be related to the feature so that other people who read the code can understand its purpose.

Reply:
Made an attempt at that now:) Sorry for the delay!